code/modules/projectiles/projectile/beams.dm

```diff
@@ -8,6 +8,9 @@
 	flag = "laser"
 	eyeblur = 2
 
+/obj/item/projectile/beam/hosbeam
+	damage = 25

```

Comment:
Because I was just finishing my SM Generator PR.

Do you honestly think I would lie? If so I'm disappointed.